vue/sbc-common-components/src/components/SbcHeader.vue

```diff
@@ -287,7 +287,11 @@
         </v-btn>
       </div>
     </v-container>
+    <div id="browse-version-alert">
+      <BrowserVersionAlert />
```

Comment:
Where is the existing banner code? Would it make sense to put this in the same place?

Reply:
Not in the header. Header doesn't include the banner.

Reply:
OK.